src/Adapter/MySQL.php

```diff
@@ -218,6 +221,7 @@ protected function getFieldMapping()
                 'fieldName' => 'name',
                 'joinCondition' => '(p.id_manufacturer = m.id_manufacturer)',
                 'joinType' => self::LEFT_JOIN,
+                'dependencyField' => 'id_manufacturer',
```

Comment:
Safety precatution, I want to add the select dynamically, and if id_manufacturer is not in initial population, this mapping would not work, because the adapter doesn't know what `(p.id_manufacturer = m.id_manufacturer)` is.